app/assets/javascripts/discourse/templates/components/group-flair-inputs.hbs

```diff
@@ -35,13 +35,13 @@
 
   <div class="avatar-flair-preview">
     <div class="avatar-wrapper">
-      <img width="45" height="45" src={{demoAvatarUrl}} class="avatar actor">
+      <img width="45" height="45" src={{demoAvatarUrl}} class="avatar actor" alt="" role="presentation">
```

Comment:
Should we add a value here? "sample avatar"

Reply:
I did this because there is no alt text on regular post avatars.